src/app/public/modules/sky-tree-view/sky-tree-view.component.ts

```diff
@@ -0,0 +1,52 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  ContentChild,
+  Input,
+  ViewEncapsulation
+} from '@angular/core';
+
+import {
+  TreeComponent
+} from 'angular-tree-component';
+
+import {
+  SkyTreeViewOptions
+} from './types/sky-tree-view-options';
+
+@Component({
+  selector: 'sky-tree-view',
+  templateUrl: './sky-tree-view.component.html',
+  encapsulation: ViewEncapsulation.None,
```

Comment:
This is fairly dangerous; see my comment above about the global style sheet.

Reply:
Fixed. See previous comments.